### PR TITLE
- fixed setSelects when disable items are on the list

### PR DESCRIPTION
**Pull request by alexisgayte**
Hi matte,

We are using your multiple-select code, and have found a bug due to disable items are not updated.
I have changed the code to make it works. Apparently it was managed every where except here.

Could you accept the pull request or let me know wha you want from me to make it approved.

Cheers,
Alexis GAYTE

### CODE REVIEW

**Comment by wenzhixin** [comment]
I have updated many codes in the latest version, can you show the bug and rebase your PR? Thanks!

**Comment by alexisgayte** [comment]
Hi,
It wasn t fixed by the changes, I rebase/merge and pull request again.
Alexis

File: docs/css/base.css [hunk 1]
```diff
@@ -1,137 +1,137 @@
...
+.pr15{padding-right:15px}
+.pr20{padding-right:20px}
+.pr30{padding-right:30px}
+.pr50{padding-right:50px}
+.pr100{padding-right:100px}
+/*颜色*/
+.bcw{background-color:#ffffff}
+.colorred{color:#ff0000}
+div.disabled{color:#cccccc}
+.button_pointer{cursor:pointer}
```
Comment (wenzhixin) [inline docs/css/base.css]: It seems that this file is not need to change.

File: multiple-select.js [hunk 2]
```diff
@@ -480,11 +480,13 @@
...
             var that = this;
             this.$selectItems.prop('checked', false);
+            this.$disableItems.prop('checked', false);
             $.each(values, function (i, value) {
                 that.$selectItems.filter(sprintf('[value="%s"]', value)).prop('checked', true);
+                that.$disableItems.filter(sprintf('[value="%s"]', value)).prop('checked', true);
             });
             this.$selectAll.prop('checked', this.$selectItems.length ===
-                this.$selectItems.filter(':checked').length);
+                this.$selectItems.filter(':checked').length + this.$disableItems.filter(':checked').length));
```
Comment (wenzhixin) [inline multiple-select.js]: I test your code in jsfiddle: http://jsfiddle.net/wenyi/knbdm5hu/28/, but it cause an error.
this.$disableItems.filter(':checked').length**)**); --> error here

Can you update the jsfiddle to test your code and show the problem you want to solve?

**Pull request closed without merging**